app/javascript/mastodon/reducers/settings.js

```diff
@@ -21,20 +21,23 @@ const initialState = ImmutableMap({
   notifications: ImmutableMap({
     alerts: ImmutableMap({
       follow: true,
+      profile_change: true,
```

Comment:
I think you also need to add the key [here](https://github.com/tootsuite/mastodon/blob/master/app/javascript/mastodon/reducers/push_notifications.js#L7).

Reply:
Thanks, I'll do so when I manage to get “normal” notifications working like I want first.

Reply:
Sure thing. On the same note, [this](https://github.com/tootsuite/mastodon/blob/master/app/models/web/push_subscription.rb#L27) may also need to be changed, since `data['alerts'][notification.type.to_s]` may not exist.

Reply:
Wouldnt that return nil, thus a false-y value, which is the default anyway?

Reply:
Assuming ActiveRecord handles `:json` fields as `Hash` objects, this should be fine: https://ruby-doc.org/core-2.2.0/Hash.html#method-i-5B-5D

Also didn't run into any error while trying it out.